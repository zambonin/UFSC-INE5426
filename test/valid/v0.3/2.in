int i=0
float f=0

f = 12 + i - f
i = [int] ([int] 12.3) / f

