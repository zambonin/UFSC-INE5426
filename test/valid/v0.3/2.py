exec(open('../../../src/scope_manager.py', 'r').read())
i = 0
f = float(0)
f = (float((12 + i)) - f)
i = int((float(int(12.3)) / f))
