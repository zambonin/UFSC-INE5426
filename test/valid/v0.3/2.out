int var: i = 0
float var: f = [float] 0
= f - [float] + 12 i f
= i [int] / [float] [int] 12.3 f
